include/deal.II/base/utilities.h

```diff
@@ -452,7 +452,7 @@ namespace Utilities
    */
   template <typename T>
   std::vector<char>
-  pack(const T &object);
+  pack(const T &object, const bool allow_compression = true);
```

Comment:
please adjust documentation to say what `allow_compression` means.

same below.